RELEASE_NOTES.md

```diff
@@ -0,0 +1,85 @@
+
+In support of these efforts, we have added functionality in Regen Ledger v4.0 to support dynamic batch minting that enables bridged assets from the same vintage to be minted to a pre-existing credit batch. Each credit batch will be "sealed" by default so that credit batches with credits issued natively on Regen Ledger can remain immutable.
+
+When credits are bridged from Regen to Polygon, the credits will be cancelled, indicating that the credits have moved to another chain or registry. The bridge service will then read the event emitted from the execution of the bridge message and process the bridge request.
+
+The functionality to support bridging assets is included in Regen Ledger v4.0 but the bridge service itself will be launched separately.
+
+## Improvements
+
+### Improved Storage
+
+Regen Ledger v4.0 makes use of an ORM storage model implemented within the orm module within Cosmos SDK that acts as an abstraction layer over the existing KV store. The orm module enables the creation of database tables with primary and secondary keys. This abstraction layer provides support for efficient lookups and will improve the velocity of feature development.
+
+### Improved API Naming
+
+Regen Ledger v4.0 includes a significant number of minor API changes intended to provide more consistent naming throughout the project and to provide an overall better user experience. The API is defined in proto files that are now available on [Buf Schema Registry](https://buf.build/regen/regen-ledger).
+
+## Additional Changes
+
+### Credit Batch Denoms
+
+Adding support for on-chain projects required updating the format of the credit batch denom to include the project ID. The credit batch denom was previously formatted to include the credit type abbreviation, the credit class ID, the start and end dates for the monitoring period, and the credit batch sequence number scoped to the credit class. The credit batch denom is now formatted to include the project ID and the credit batch sequence number is now scoped to the project.
+
+An example of a credit batch in Regen Ledger v3.0:
+
+```
+C01-20200101-20210101-001
+```
+
+*The first credit batch from the first credit class of the "C" credit type with a start date of January 1st 2020 and an end date of January 1st 2021 (credit class id "C01" and batch sequence "001").*
+
+An example of a credit batch in Regen Ledger v4.0:
+
+```
+C01-001-20200101-20210101-001
+```
+
+*The first credit batch from the first project of the first credit class of the "C" credit type with a start date of January 1st 2020 and an end date of January 1st 2021 (credit class id "C01", project id "C01-001", and batch sequence "001").*
+
+## Changelog
+
+For a full list of changes since Regen Ledger v3.0, please see [CHANGELOG.md](./CHANGELOG.md).
+
+## Validator Upgrade Guide
+
+An upgrade guide for validators is available at [Upgrade Guide v4.0](https://docs.regen.network/validators/upgrades/v4.0-upgrade.html).
```

Comment:
@ryanchristo is this supposed to 404?

Reply:
Approved - assuming we'll create this later.

Reply:
It has yet to be written but this will be the correct link: https://github.com/regen-network/regen-ledger/issues/1238